benchkit/communication/__init__.py

```diff
@@ -702,6 +703,10 @@ def remote_host(self) -> str | None:
     def is_local(self) -> bool:
         return False
 
+    @property
+    def get_ipaddress(self) -> str:
+        return self._ssh_host_info["hostname"]
```

Comment:
that is not necessarily an IP address. Do you need IP addresses or hosts?

Reply:
Modify this to use the SSH_CONNECTION environment variable to get the IP address

Reply:
Added a README instruction and removed the *.jar file. Is that following the structure you proposed? Could you check that the instructions work on your side as well?